snuba/snuba_migrations/spans/0004_spans_group_raw_col.py

```diff
@@ -0,0 +1,58 @@
+from typing import Sequence
+
+from snuba.clickhouse.columns import Column, UInt
+from snuba.clusters.storage_sets import StorageSetKey
+from snuba.migrations import migration, operations
+from snuba.migrations.operations import OperationTarget, SqlOperation
+
+storage_set_name = StorageSetKey.SPANS
+local_table_name = "spans_local"
+dist_table_name = "spans_dist"
+
+
+class Migration(migration.ClickhouseNodeMigration):
+    """
+    Adds group_raw columns to store the scrubbed group hash.
```

Comment:
Nit: `group_raw` would store the hash of the raw description.

Reply:
fixed